app/(website)/juguete/[slug]/page.tsx

```diff
@@ -11,16 +14,35 @@ interface Props {
 
 async function Page({ params: { slug } }: Props) {
   const toy = await client.fetch<Toy>(toySlugQuery, { slug });
-
+  const images = toy.images;
+  const description = toy.description[0];
   return (
     <section className="py-10">
-      <div className="mx-auto max-w-5xl">
-        <h2 className="mb-8 text-center text-2xl font-bold md:text-4xl">
+      <div className="mx-auto max-w-3xl">
+        <h4 className="mb-4 text-center">Juguetes</h4>
+        <h1 className="mb-8 text-center text-2xl font-bold md:text-4xl">
           {toy.name}
-        </h2>
-        <div>
-          <LoanRequestForm />
+        </h1>
+        <div className="relative mx-auto mb-10 lg:max-w-3xl">
+          <Carousel isFullWidth>
+            {images.map((image) => (
+              <Image
+                width={0}
+                height={0}
+                sizes="(max-width: 768px) 100vw"
+                placeholder={"blur"}
+                blurDataURL={image.placeholder}
+                key={image.asset._ref}
+                src={urlFor(image).url()}
+                alt={image.altText}
+                className="mx-auto aspect-video w-full rounded"
+              />
+            ))}
+          </Carousel>
         </div>
```

Comment:
I was looking at the design, we may need to get the Carousel out of the container (`section` > `div`)

![image](https://github.com/Juguetear/juguetear-web/assets/78808163/b62c7224-ffd5-497b-9831-7a66f30efc02)

Reply:
@alexxuss1996 I was thinking we can wrap main headings (Juguetes, Winnie Pooh) on a container with a `max-w-3xl` (768px).
And then **only** the Carousel will be `max-w-5xl` (1024px), then the rest will be on another container like the headings `max-w-3xl` (768px).